plugins/inputs/jenkins/jenkins_test.go

```diff
@@ -97,8 +97,8 @@ func (h mockHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusNoContent)
 		return
 	}
-	// Ignore the returned error as the tests will fail anyway
-	//nolint:errcheck,revive
+
+	//nolint:errcheck,revive // ignore the returned error as the tests will fail anyway
 	w.Write(b)
```

Comment:
Sometimes the nolint is above, sometimes in-line. Personally I prefer above so we avoid really long lines or line length concerns and I can clearly see what is ignored. Is there a standard or both work?

Reply:
Both work, I prefer in-line for short lines - it is much easier to see in search view in IDEs for what operation `//nolint`s were enabled and why.
I corrected this line.